students/roviram/lesson07/html_render.py

```diff
@@ -0,0 +1,301 @@
+# Student: Miguel Rovira-Gonzalez
+"""
+PLEASE NOTE: This is not my code, I did not learn anything from this assignment and it did not teach me about Classes and Inheritance
+I got stuck on step 5.
+This was extremely difficult for a beginner programmer.
+"""
```

Comment:
This assignment is definitely a tough one. Lets try to go over this code together or with the Professor and we can try to work through it. I would highly recommend Office Hours to review this assignment / get a better understanding of Classes/Objects/OOP.